packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -281,6 +283,18 @@ class PaginatedDataTable extends StatefulWidget {
   /// {@macro flutter.widgets.scrollable.dragStartBehavior}
   final DragStartBehavior dragStartBehavior;
 
+  ///Defines the color of header background
+  ///
+  final Color? headerBackgroundColor;
+
+  ///Defines the color of footer background
+  ///
+  final Color? footerBackgroundColor;
+
+  ///Defines the footer text style
+  ///
+  final TextStyle footerStyle;
```

Comment:
This documentation should be in full sentences, and have more information pertaining to defaults and theming. See other property docs for examples.